pulsar-broker/src/test/java/org/apache/pulsar/broker/namespace/OwnerShipCacheForCurrentServerTest.java

```diff
@@ -63,4 +68,21 @@ public void testOwnershipForCurrentServer() throws Exception {
                     namespaceServices[i].getOwnerAsync(bundle).get().get().getNativeUrl());
         }
     }
+
+    @Test(timeOut = 30000)
+    public void testCreateTopicWithNotTopicNsOwnedBroker() {
+        String topicName = "persistent://" + TENANT + "/" + NAMESPACE + "/" + "testCreateTopic";
```

Comment:
can we take this opportunity of touching this test to not use here and in the test above a fixed name for the test ?
we should have somewhere an utility to generate a random name

Reply:
Good suggestion. Addressed.